aequilibrae/project/database_specification/transit/tables/periods.sql

```diff
@@ -0,0 +1,22 @@
+--@ The periods table holds the time periods and their period_id. Default entry with id 1 is the entire day.
+--#
+CREATE TABLE if not exists periods (period_id               INTEGER UNIQUE NOT NULL,
```

Comment:
This table should go in the project database, as these periods are used for everything

Reply:
I've just move the `periods` table into the Project database. I've also created a small wrapper class around it, similar to the existing Nodes, Links, and Zones classes.